pkg/uevent/queue.go

```diff
@@ -119,15 +103,44 @@ func (e *eventQueue) push(event *deviceEvent) {
 				e.keyCh <- event.devPath
 			},
 		)
-	} else {
-		e.keyCh <- event.devPath
 	}
+
 	e.events[event.devPath] = event
+	return existingEvent, found
+}
+
+func (e *eventQueue) push(event *deviceEvent) {
+	existingEvent, found := e.set(event)
```

Comment:
tabwidth should be 4 spaces across the project. Everyone should standardize on this. For now, please set it to 4 spaces. We can add a separate PR to add a linter to check for this